### PR TITLE
Bump harvester cloud provider 0.1.6 and csi driver 0.1.7

### PR DESCRIPTION
**Pull request by yaocw2020**
chart changes: https://github.com/harvester/charts/pull/58/files

### CODE REVIEW

**Review by guangbochen — APPROVED** [review]

**Review by jiaqiluo — CHANGES_REQUESTED** [review]
Could you delete the existing `harvester-cloud-provider-100.0.0+up0.1.5` and `harvester-csi-driver-100.0.0+up0.1.6` from the charts/, assets/, and the index.yaml file?  They are replaced by the new versions introduced in this PR so we need to remove them. 

This is a general rule that we should remove the old version when introducing a new version in the dev branch.

**Comment by yaocw2020** [comment]
> Could you delete the existing `harvester-cloud-provider-100.0.0+up0.1.5` and `harvester-csi-driver-100.0.0+up0.1.6` from the charts/, assets/, and the index.yaml file? They are replaced by the new versions introduced in this PR so we need to remove them.
> 
> This is a general rule that we should remove the old version when introducing a new version in the dev branch.

I have deleted the old version.

File: charts/harvester-cloud-provider/harvester-cloud-provider/100.0.0+up0.1.6/Chart.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ annotations:
   catalog.cattle.io/kube-version: '>= 1.18'
   catalog.cattle.io/namespace: kube-system
   catalog.cattle.io/os: linux
-  catalog.cattle.io/rancher-version: '>= 2.6.1'
+  catalog.cattle.io/rancher-version: '>= 2.6.1.0 <= 2.6.99-0'
```
Comment (jiaqiluo) [inline charts/harvester-cloud-provider/harvester-cloud-provider/100.0.0+up0.1.6/Chart.yaml:8]:
```suggestion
  catalog.cattle.io/rancher-version: '>= 2.6.1-0 <= 2.6.99-0'
```

File: charts/harvester-csi-driver/harvester-csi-driver/100.0.0+up0.1.7/Chart.yaml [hunk 2]
```diff
@@ -3,12 +3,12 @@ annotations:
   catalog.cattle.io/display-name: Harvester CSI Driver
   catalog.cattle.io/experimental: "true"
   catalog.cattle.io/kube-version: '>= 1.18'
+  catalog.cattle.io/namespace: kube-system
   catalog.cattle.io/os: linux
-  catalog.cattle.io/rancher-version: '>= 2.6.1'
+  catalog.cattle.io/rancher-version: '>= 2.6.1.0 <= 2.6.99-0'
```
Comment (jiaqiluo) [inline charts/harvester-csi-driver/harvester-csi-driver/100.0.0+up0.1.7/Chart.yaml:8]:
```suggestion
  catalog.cattle.io/rancher-version: '>= 2.6.1-0 <= 2.6.99-0'
```

File: charts/harvester-csi-driver/harvester-csi-driver/100.0.0+up0.1.7/Chart.yaml [hunk 3]
```diff
@@ -3,12 +3,12 @@ annotations:
...
   catalog.cattle.io/release-name: harvester-csi-driver
   catalog.cattle.io/ui-component: harvester-csi-driver
   catalog.cattle.io/upstream-version: 0.1.6
```
Comment (jiaqiluo) [inline charts/harvester-csi-driver/harvester-csi-driver/100.0.0+up0.1.7/Chart.yaml:11]:
```suggestion
  catalog.cattle.io/upstream-version: 0.1.7
```

File: charts/harvester-cloud-provider/harvester-cloud-provider/100.0.0+up0.1.6/Chart.yaml [hunk 4]
```diff
@@ -5,7 +5,7 @@ annotations:
...
   catalog.cattle.io/release-name: harvester-cloud-provider
   catalog.cattle.io/ui-component: harvester-cloud-provider
   catalog.cattle.io/upstream-version: 0.1.5
```
Comment (jiaqiluo) [inline charts/harvester-cloud-provider/harvester-cloud-provider/100.0.0+up0.1.6/Chart.yaml:11]:
```suggestion
  catalog.cattle.io/upstream-version: 0.1.6
```

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

**Review by jiaqiluo — APPROVED** [review]

**Comment by guangbochen** [comment]
@jiaqiluo can u please also help to merge this PR thanks?

**Pull request merged**